waisntechtools/alerts/Twilio/fetch_sms.py

```diff
@@ -0,0 +1,29 @@
+from twilio.rest import Client
+from django.conf import settings
+from django.http import HttpResponseForbidden
+from twilio.request_validator import RequestValidator
+
+import requests
+import os
+
+
+def fetch_twilio(request):
+    account_sid = os.environ.get('TWILIO_ACCOUNT_SID')
+    auth_token = os.environ.get('TWILIO_AUTH_TOKEN')
+
+    validator = RequestValidator(auth_token)
+    request_valid = validator.validate(
+        request.build_absolute_uri(),
+        request.POST,
+        request.META.get('HTTP_X_TWILIO_SIGNATURE', ''))
+
+    if request_valid or settings.DEBUG:
```

Comment:
This seems weird to me. If the request is invalid, even in debug mode, does it make sense to perform the request?

Reply:
This was a mistake.  Copied from Twilio docs and forgot to remove "debug" part